portfolio/src/main/java/com/google/sps/servlets/DeleteSomeDataServlet.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import java.util.Date;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.text.*;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/delete-some-data")
+public class DeleteSomeDataServlet extends HttpServlet {
```

Comment:
Another option would be to have a single servlet do two actions.  This would be the better approach if your delete servlet had heavier business logic, since you wouldn't want to duplicate code if possible.

As to how to do this, you can:
1.  Assume blank or missing id param for delete all, kinda dangerous though if someone mistypes the url
2.  Assume id=-1 for delete all
3.  Add another param to signal what to do, something like "action=delete_all" or "op=delete_all" or "delete_all=true"